setup.py

```diff
@@ -7,7 +7,7 @@
 setup(
     ext_modules=[
         Extension(
-            "_mt2",
+            "mt2._mt2",
```

Comment:
```suggestion
            "_mt2",
```

The problem was that the `mt2` directory was missing, not that this was in the wrong place.